.changeset/puny-lies-fall.md

```diff
@@ -0,0 +1,13 @@
+---
+"@fluidframework/agent-scheduler": minor
+"@fluid-experimental/azure-scenario-runner": minor
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore-definitions": minor
+"@fluidframework/odsp-driver": minor
+"@fluidframework/ordered-collection": minor
+"@fluidframework/test-runtime-utils": minor
+---
+
+Use telemetry logger base type since IFluidDataStoreRuntime is public
+
+renaming some other tyes to align on the "Base" nomenclature for types used in public API surface
```

Comment:
This is really what we need to communicate in the release notes. All the other changes are internal to Fluid, or don't affect its consumers.
```suggestion
"@fluidframework/datastore-definitions": major
---

IFluidDataStoreRuntime.logger is now an ITelemetryBaseLogger

`IFluidDataStoreRuntime.logger` is now an `ITelemetryBaseLogger` instead of the deprecated `ITelemetryLogger`. The `sendTelemetryEvent()`, `sendErrorEvent()`, or `sendPerformanceEvent()` methods were not intended for users of `IFluidDataStoreRuntime`. You can keep using the logger's `send()` method to generate telemetry.
```